msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.Common.targets

```diff
@@ -777,6 +777,12 @@ Copyright (C) 2013-2016 Xamarin. All rights reserved.
 			<_CodesignDisableTimestamp Condition="'$(_SdkIsSimulator)' == 'true' Or '$(_BundlerDebug)' == 'true'">True</_CodesignDisableTimestamp>
 		</PropertyGroup>
 
+		<!-- Delete any crash dumps in the app bundle that might exist. Ref: https://github.com/xamarin/xamarin-macios/issues/12320 -->
+		<ItemGroup>
+			<_MonoCrashDumpsInAppBundle Include="$(AppBundleDir)/mono_crash.mem.*" />
```

Comment:
Those files will never exist on Windows because the real app bundle is on the Mac, I think you need a task for this so it can be executed remotely.

Reply:
Fixed to use a task to collect the files.